src/auto_scheduler/search_policy/sketch_policy.cc

```diff
@@ -218,15 +218,45 @@ State SketchPolicyNode::Search(int n_trials, int early_stopping, int num_measure
   }
 }
 
-Array<State> SketchPolicyNode::SearchOneRound(int num_random_states, Array<State>* random_states) {
-  // Temporal object to be used if the input pointer is nullptr
-  Array<State> temp_random_states;
-  if (random_states == nullptr) {
-    random_states = &temp_random_states;
-  } else {
-    random_states->clear();
+std::pair<Array<MeasureInput>, Array<MeasureResult>> SketchPolicyNode::ContinueSearchOneRound(
```

Comment:
From the functionality it doesn't look like "continue" search one round, but more like `SearchOneRound`. On the other hand, the current search one round it is calling is more like `TuneStates`.

Reply:
I still prefer the current names.